curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-288-an-enormous-factorial.md

```diff
@@ -8,37 +8,41 @@ dashedName: problem-288-an-enormous-factorial
 
 # --description--
 
-For any prime p the number N(p,q) is defined by
+For any prime $p$ the number $N(p,q)$ is defined by $N(p,q) = \sum_{n=0}^q T_n \times p^n$ with $Tn$ generated by the following random number generator:
```

Comment:
The `q` should refer to the upper bound of the summation. As in sum `T_n x p^n`, for `n` values from `n = 0` to `n = q`. Some other ways to write it:
![image](https://user-images.githubusercontent.com/60067306/126287477-34395875-989e-4ba5-90cd-9fb925fcdcb6.png)
![image](https://user-images.githubusercontent.com/60067306/126289769-e6338525-3ff9-42a0-986d-809002ccfb3a.png)
![image](https://user-images.githubusercontent.com/60067306/126289855-9a3dda69-926f-4095-a6df-44a0d7625228.png)

I guess `to q` also could be used, but I think this is one of the places where it's good idea to improve the notation (while still keeping the same meaning), rather than sticking to original.

In suggestion it is looking more as power indication. But power of what? The whole sum? In such case something like this might be more appropriate:
![image](https://user-images.githubusercontent.com/60067306/126288595-204e4b98-3a8c-40c4-9673-b87f951d505e.png)
But I don't think that's a case here, as it still would need to have some upper bound.

Some reference: https://en.wikipedia.org/wiki/Summation

Reply:
Ahh - yea, I thought the `q` was supposed to be an exponent. So is the way you had it accurate then? If so, I can revert the suggestion to that. Or else I can change it to whatever you want. I think I personally prefer your third example:
![image](https://user-images.githubusercontent.com/60067306/126289855-9a3dda69-926f-4095-a6df-44a0d7625228.png)

Reply:
Yeah, as far as I know the way it is right now is correct too. I don't really mind changing it to different option, so if there won't be more voices, I'll change it tomorrow, or the day after it, to the third example, as you like.

Reply:
> as far as I know the way it is right now is correct too.
I am taking that as meaning the way you originally had it. So I reverted my suggestion to that in case you just want to go with it.